script/Arcueid/Day 3/03_00_ARC03_2B.txt

```diff
@@ -314,7 +314,7 @@ The character for 'seven', and next to it, the character for 'night'.
 -- Page 107, Offset 4226.
 -- 七つ夜、と書かれているだけよ」
 // 七つ夜、と書かれているだけよ」
-It just says 'Nanatsu-yoru'."
+It just says %{i}Nanatsu-yoru%{/i}."
```

Comment:
Wouldn't that mean you'll need to italicize anything written on any object?

Reply:
Could go either way on this one since it's a fairly important name

Reply:
> Wouldn't that mean you'll need to italicize anything written on any object?

Not sure this follows. But like Ross said, this is an important name that's not being localized.